robot.py

```diff
@@ -30,6 +30,7 @@ def disabledInit(self):
     def disabledPeriodic(self):
         """This code runs every 20ms when the robot is disabled."""
         super().disabledPeriodic()
+        Command.getRobot().drivetrain.diffdrive(0, 0)
```

Comment:
This technically works, but I have a few comments. First, the function `Command.getRobot()` returns the robot instance, which in this case is equivalent to `self`. We only need to use `Command.getRobot()` when we are not in `robot.py`. The rest of its works fine, but it is not the most intuitive solution and could introduce future bugs. For instance, if we in the future (as is likely) add ramping to the diffdrive function (where we don't drop immediately to zero but scale down the input over time), then when we disable the robot it will continue forward for a short time. The better solution would be to add a zero method to the drivetrain subsystem which would call `_left_master_talon.set(0)` and `_right_master_talon.set(0)`. Then, instead of calling `drivetrain.diffdrive(0,0)`, you can call `drivetrain.zero()`.